day1/adventcode2015_day1_refactor.py

```diff
@@ -0,0 +1,32 @@
+with open('input.txt', 'r') as floors:
+    visited_floors: str = floors.readline().strip()
+
+
+def count_by_character(input_string, counter=0):
```

Comment:
counter should be initialized inside function, not passed as parameter

Reply:
Because it shouldn't be changable as it should always be 0?

Reply:
no, because on each function invocation 'parent' function needs to provide new counter, and it's not it's responsibility